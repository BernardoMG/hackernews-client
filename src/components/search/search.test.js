/* eslint-disable no-undef */
import React from 'react'
import ReactDOM from 'react-dom'
import Renderer from 'react-test-renderer'
import Search from './search.js'

describe('Search', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Search>Search</Search>, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  test('has a valid snapshot', () => {
    const component = Renderer.create(<Search>Search</Search>)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})

