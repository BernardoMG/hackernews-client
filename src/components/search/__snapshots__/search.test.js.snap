// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search has a valid snapshot 1`] = `
<form>
  Search
  <input
    type="text"
  />
  <button
    className="button"
    onClick={null}
    type="submit"
  >
    Search
  </button>
</form>
`;
